querymonitor/querymonitor.pl

```diff
@@ -673,6 +691,13 @@ sub new{
 	for(qw|API_TOKEN YBA_HOST CUST_UUID UNIV_UUID|){
         $opt{$_} or die "ERROR: Required parameter --$_ was not specified.\n";
     }
+	for(qw|API_TOKEN  CUST_UUID UNIV_UUID|){
+        (my $value=$opt{$_})=~tr/-//d; # Extract and zap dashes
+		my $len = length($value);
+		next if $len == 32; # Expecting these to be exactly 32 bytes 
+        warn "WARNING: Expecting 32 valid bytes in Option $_=$opt{$_} but found $len bytes. \n";
+		sleep 2;
```

Comment:
Why sleep?

Reply:
To give them a chance to read the warning message.

There will be more errors coming after this - did not want this message to get buried unnoticed.